pkg/action/server.go

```diff
@@ -32,38 +32,57 @@ func Server(cfg *config.Config, logger log.Logger) error {
 		"go", version.Go,
 	)
 
-	ctx := context.WithValue(
-		context.Background(),
-		oauth2.HTTPClient,
-		&http.Client{
-			Transport: &http.Transport{
-				TLSClientConfig: &tls.Config{
-					InsecureSkipVerify: cfg.Server.Insecure,
-				},
-			},
-		},
+	var (
+		client *github.Client
 	)
 
-	client, err := github.NewEnterpriseClient(
```

Comment:
I was surprised that this previously only had `NewEnterpriseClient`, and not also a `NewClient` reference.

Admittedly I did not test my changes against github.com 😎

Reply:
Yeah I had a default value for the github.com API, but I found out it doesn't find any of my public repositories.